### ISSUE TITLE
Ember.js Community Color Stack

### ISSUE THREAD

**Issue by nathanhammond** [ember-a11y/issues#1]
We need a color stack that is going to meet the needs of both form and function and (hopefully) apply across many Ember web properties. Presumably the only colors we can't change are the ones in the logos, but we should probably aim to darken/lighten existing theme colors so that we maintain branding.

Unfortunately, [we didn't factor out all color references into variables](https://github.com/emberjs/website/tree/master/source/stylesheets) for the website, so it's hard for us to grab a collection of colors and just redefine them. [A recent pass for creating an ember styleguide is closer and probably where we want to start from.](https://github.com/locks/ember-styleguide/blob/master/app/styles/base/_variables.scss)

Logo colors:
- `#E34C32` (orange)
- `#612116` (brown)
- `#FFFAF1` (off-white) [(the ember-data-on-dark logo uses pure white, mistake?)](http://emberjs.com/images/brand/ember_Data-Dark.png)

Tracked issues:
- https://github.com/ember-a11y/ember-a11y/pull/27
- https://github.com/locks/ember-styleguide/issues/8
- https://github.com/emberjs/website/issues/2613

**Comment by acorncom** [comment]
Has there been any progress on this?  I may be able to find volunteers to do the legwork if we can come up with a color stack we all agree on ...

**Comment by MelSumner** [comment]
I think that the brown is really supposed to be #632116- see the image attached to this. 
The first part is the orange as shades, the second is orange as complementary, and the third custom one integrates the dark brown and the off-white. 
@acorncom @nathanhammond LMK what you think. 
![ember-colorscheme](https://cloud.githubusercontent.com/assets/4587451/18999276/5f9f65ac-8702-11e6-8617-af7a7bdb0fee.jpg)

**Comment by BrianSipple** [comment]
@nathanhammond You're correct on the question of the `ember-data` logo white being... um... white.

As you also highlighted, though -- at least the way I understood it -- we don't have any "official" colors yet. So it would be important to keep in mind that we aren't _strictly_ tied to a specific hex code (hence the nice off-white in @MelSumner's third palette). Perhaps I'm stating the obvious -- apologies if so 😄. 

Another suggestion I'd like to offer is not necessarily limiting ourselves to just one primary/secondary color combination. 

Our hues of orange would be the mainstay, for sure. But for example, I also think [Ember Fastboot](https://www.ember-fastboot.com/) did an awesome job with their color scheme, and it shows how there's great power and potential to be had if we established a few (like, one or two... or three) _alt_-primary colors. They'd still have their own A11y-compliant compliments, but it would allow for different projects/sites to create aesthetics that look and feel distinct, but are still on-brand.

**Comment by MelSumner** [comment]
Well, we have been playing with the coral/navy/mint combo for a while, but I think if we were to do that, it might end up dating us, which is why I am hesitant to do that. I think if we stick with shades/complementary colors we could benefit from an approach that has a more timeless appeal.
